app/src/main/res/layout/error_retry.xml

```diff
@@ -27,6 +27,7 @@
         android:textAlignment="center"
         android:textAllCaps="true"
         android:textAppearance="@style/TextAppearance.AppCompat.Body1"
+        android:background="?attr/colorPrimary"
```

Comment:
Why was this added? I don't think it is needed after @B0pol introduced `ServiceColoredButton`